src/app/shell/personal-cabinet/parent/create-child/child-form/child-form.component.ts

```diff
@@ -17,11 +20,17 @@ export class ChildFormComponent implements OnInit {
 
   @Output() deleteForm = new EventEmitter();
 
-  today: Date = new Date();
+  dateFilter: RegExp;
```

Comment:
you can define variables directly in declaration section

Reply:
have updated this part, thanks